apps/condo/domains/user/schema/RegisterNewUserService.js

```diff
@@ -1,31 +1,29 @@
 const { GQLCustomSchema } = require('@core/keystone/schema')
-const { REGISTER_NEW_USER_MESSAGE_TYPE } = require('@condo/domains/notification/constants')
+const { REGISTER_NEW_USER_MESSAGE_TYPE, SMS_TRANSPORT, EMAIL_TRANSPORT } = require('@condo/domains/notification/constants')
 const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
 const { COUNTRIES } = require('@condo/domains/common/constants/countries')
 const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
 const { MIN_PASSWORD_LENGTH_ERROR } = require('@condo/domains/user/constants/errors')
 const { MIN_PASSWORD_LENGTH } = require('@condo/domains/user/constants/common')
 const { ConfirmPhoneAction: ConfirmPhoneActionServerUtils, User: UserServerUtils } = require('@condo/domains/user/utils/serverSchema')
-const isEmpty = require('lodash/isEmpty')
 const {
     CONFIRM_PHONE_ACTION_EXPIRED,
 } = require('@condo/domains/user/constants/errors')
+const { STAFF } = require('@condo/domains/user/constants/common')
+const { isEmpty } = require('lodash')
 
 async function ensureNotExists (context, field, value) {
-    if (isEmpty(value)) {
-        throw new Error(`[error] Unable to check field ${field} uniques because the passed value is empty`)
-    }
-    const existed = await UserServerUtils.getAll(context, { [field]: value, type: 'staff' })
+    const existed = await UserServerUtils.getAll(context, { [field]: value, type: STAFF })
     if (existed.length !== 0) {
         throw new Error(`[unique:${field}:multipleFound] user with this ${field} is already exists`)
     }
 }
-
+// TODO(zuch): create registerStaffUserService, separate logic of creating employee, make confirmPhoneActionToken to be required, remove meta, args to UserInput
 const RegisterNewUserService = new GQLCustomSchema('RegisterNewUserService', {
     types: [
         {
             access: true,
-            type: 'input RegisterNewUserInput { dv: Int!, sender: SenderFieldInput!, name: String!, email: String!, password: String!, confirmPhoneActionToken: String, phone: String, meta: JSON }',
+            type: 'input RegisterNewUserInput { dv: Int!, sender: SenderFieldInput!, name: String!, email: String, password: String!, confirmPhoneActionToken: String, phone: String, meta: JSON }',
```

Comment:
phone is required?!

Reply:
This mutation needs to be separated as it is used in several scenarios. 
1. Registering new user (in this case phone is not required as we pass confirm phone action as an argument)
2. Invite employee - in this case phone is required